src/evaluation.py

```diff
@@ -0,0 +1,79 @@
+# src/evaluation.py
+
+import logging
+import matplotlib.pyplot as plt
+from sklearn.metrics import classification_report, accuracy_score
+
+logger = logging.getLogger(__name__)
+logging.getLogger("matplotlib.category").setLevel(logging.WARNING)
+
+
+def evaluate_model(model, x_test, y_test) -> plt.Figure:
```

Comment:
Missing type annotations. For model, this could be hard. I see three options:

1. Just using Any, but then the developer has to pay attention.
2. Creating an abstract base class specifying the needed methods, i.e. predict. This would be similar to sklearn.
3. Passing a callable method for prediction and not the model.

Personally, I would prefer option 2.